powershell/VstsTaskSdk/ServerOMFunctions.ps1

```diff
@@ -421,6 +426,26 @@ function Get-VssHttpClient {
     }
 }
 
+function Get-WebProxy {
+    [CmdletBinding()]
+    param()
+
+    Trace-EnteringInvocation -InvocationInfo $MyInvocation
+    try
+    {
+        $ProxyUrl = Get-TaskVariable -Name Agent.ProxyUrl
+        $ProxyUserName = Get-TaskVariable -Name Agent.ProxyUserName
```

Comment:
Do you need to declare these as strings so the right constructor is always chosen below, even when these are null or empty?

E.g. `[string]$proxyUserName = Get-TaskVariable -Name Agent.ProxuUserName`